projects/vdk-plugins/vdk-jupyter/README.md

```diff
@@ -0,0 +1,40 @@
+# jupyter
+
+Prototype of a plugin for running Jupyter notebooks as vdk steps
+
+TODO: what the project is about, what is its purpose
+
```

Comment:
Are the notebooks compiled to python steps that are evaluated, or are those evaluated directly? I think this is a part of the VEP-994, that links design and implementation stories; 
Also, the relationships are bidirectional, for example the `airflow-provider-vdk` does link the VEP-554 of origin.

Reply:
Rhe notebooks are compiled to python and sql steps as the standard ones with one additional field "code" which holds the code we get from cell/cells as string.
Since it is just a prototype, I have not worked on the README that much. When doing the real plugin I will create the links. I will enhance the README and the VEP in the next few days.